draft-ietf-rats-corim.md

```diff
@@ -1254,11 +1254,28 @@ applies to all measurements in the triple, including those in `measurement-value
 # CoBOM {#sec-cobom}
 
 A Concise Bill of Material (CoBOM) object represents the signal for the
-Verifier to activate the listed tags. Data contained in a tag MUST NOT be used
+Verifier to activate the listed tags. Data contained in a tag SHOULD NOT be used
 for appraisal until a CoBOM which activates that tag has been received and
 successfully processed. All the tags listed in the CoBOM must be activated in
 the same transaction, i.e., either all or none.
 
+The number of CoBOMs required in a given supply chain eco-system is dependent on
+Verifier Owner's Appraisal Policy for Evidence, which is often driven by the complexity and nature of
+the use case.
+
+For example, in cases where only limited CoRIMs are produced by a single authority, a Verifier Owner may have
+a policy that CoBOM's are not required. In this case, signature chain over each CoRIM is sufficient authorization,
+and may not require additional authorization via CoBOM. Any tags in a CoRIM received by a Verifier
+are activated immediately and treated valid for appraisal
+
+However, a more common case is the one where Verifier receives multiple CoRIMs containing
```

Comment:
```suggestion
However, a more common case is one where the Verifier receives multiple CoRIMs containing
```

Reply:
I think, the original line 1271 does makes sense, though I should use the comma to stress the use case.

Reply:
resolved due to new text in PR